Assets/ChangeScene.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ChangeScene : MonoBehaviour {
+
+	public string targetScene;
+	public Vector3 newPos = new Vector3(0,0,0);
+
+	void OnTriggerEnter2D( Collider2D coll ) {
+		GameObject collObj = coll.gameObject;
+		Debug.Log ("rdy cg sce" + collObj.tag);
```

Comment:
Use complete sentence for Debug.log message.
Consider other developer seeing this